server/clientmessages.py

```diff
@@ -0,0 +1,660 @@
+from enum import Enum, auto
+import functools
+from typing import NamedTuple, Optional, Union
+
+
+class Target(Enum):
+    """
+    Valid entries for the `'target'` field in a message from the client.
+    """
+
+    none = 0
+    game = 1
+    connectivity = 2  # deprecated
+
+
+class ActionField(Enum):
+    """
+    Valid entries for the `'action'` field in a message from the client.
+    """
+
+    list_avatar = 1
+    select = 2
+
+
+class CommandField(Enum):
```

Comment:
Need to figure out how we can get rid of this class and generate the string to class name mapping automatically. Maybe some metaclass magic would work so that you only need to give the `command` attribute a default value.

Reply:
I think that would be straightforward, I went for this because I wanted `pdoc` to also generate a list of all possible commands. That probably also works with "metaclass magic" :)